readthedocs/search/documents.py

```diff
@@ -69,6 +69,7 @@ class PageDocument(RTDDocTypeMixin, Document):
     # Metadata
     project = fields.KeywordField(attr='project.slug')
     version = fields.KeywordField(attr='version.slug')
+    doctype = fields.KeywordField(attr='version.documentation_type')
```

Comment:
Does this require reindexing?

Reply:
yeah, but we can do it the next year, we aren't using it yet (still querying from the db or falling back to None). Tracking it here https://github.com/readthedocs/readthedocs.org/issues/7762.